server/templates/custom_dc/worldbank/homepage.html

```diff
@@ -24,17 +24,18 @@
 {% set is_hide_sub_footer = true %}
 
 {% block head %}
-<link rel="stylesheet" href={{url_for('static', filename='css/static.min.css', t=config['VERSION'])}}>
+<link rel="stylesheet" href={{url_for('static', filename='css/homepage.min.css', t=config['VERSION'])}}>
+ <script src={{url_for('static', filename='homepage_custom_dc.js', t=config['VERSION'])}} async></script>
+
 {% endblock %}
 
 {% block content %}
+<div class="container">
+  <h4>Explore World Bank Data</h4>
```

Comment:
Just want to confirm move from `<h3>` to `<h4>` is intentional.

Reply:
Yea, it feels too big with the query box..